modules/ztp-image-based-upgrade.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-talm-updating-managed-policies.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-image-based-upgrade-concept_{context}"]
+= Image-based upgrade for {sno} cluster with the {lcao}
+
+The {lcao} provides you with an alternative way to upgrade the platform version of a managed {sno} cluster.
+The image-based upgrade is faster than the standard upgrade method and allows you to directly upgrade from {product-title} <4.y> to <4.y+2>, and <4.y.z> to <4.y.z+n>.
+
+This upgrade method utilizes generated images from a dedicated seed cluster.
+A seed cluster is a managed {sno} cluster deployed and configured with the target {product-title} version and applications.
```

Comment:
Applications would generally not be part of the seed image. The seed image would include: 
- The target OCP version 
- The day 2 operators
- Configuration that is common to all target SNOs that the seed will be used for

Reply:
@browsell I revised this based on the linked section. Please let me know if the current level of detail is okay or if you want to see more.